File: Domain/Noticia.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    public class Noticia
    {
        public Guid Id { get; set; }
        public string Title { get; set; }
        public string Url { get; set; }
        public DateTime Date { get; set; }
        public string Body { get; set; }
        public ICollection<GalleryNoticia> GalleryNoticias { get; set; } = new List<GalleryNoticia>();
        public ICollection<Gallery> Galleries { get; set; } = new List<Gallery>();
        public string AppUserId {get; set;}
        public AppUser AppUser {get; set;}
    }
}

File: client-app/src/app/models/noticia.ts
import { Gallery } from "./gallery";
import { ImageDto } from "./image";

export interface Noticia {
    id: string;
    title: string;
    url: string;
    date: Date;
    body: string;
    portraitUrl?: string;
    portrait?: ImageDto;
    appUserId: string;
    galleries?: Gallery[] | undefined;
}
export class Noticia implements Noticia {
    constructor(init?: Noticia) {
        Object.assign(this, init);
    }
}
export class NoticiaFormValues {
    id?: string = undefined;
    url: string ='';
    title: string = '';
    body: string = '';
    date: Date | null = null;
    appUserId: string = '';
    constructor(noticia?: NoticiaFormValues) {
        if (noticia) {
            this.id = noticia.id;
            this.url = noticia.url;
            this.title = noticia.title;
            this.body = noticia.body;
            this.date = noticia.date;
            this.appUserId = noticia.appUserId;
        }
    }
}